cypress/e2e/pages/mostReadPage/index.cy.js

```diff
@@ -14,52 +12,43 @@ const pageType = 'mostReadPage';
  * pidgin: LTR service
  * serbian: service with variant
  */
-const services = ['arabic', 'igbo', 'pidgin', 'serbian'];
-
-Object.values(config).forEach(
-  ({ name: service, variant, pageTypes, isWorldService }) => {
-    if (isWorldService && services.includes(service)) {
-      const { mostReadPage } = pageTypes;
-
-      const urls =
-        mostReadPage.environments &&
-        mostReadPage.environments[getAppEnv()]?.paths;
+const testSuites = [
+  {
+    path: '/pidgin/popular/read',
+    service: 'pidgin',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/arabic/popular/read',
+    service: 'arabic',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/igbo/popular/read',
+    service: 'igbo',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+  {
+    path: '/serbian/cyr/popular/read',
+    service: 'serbian',
+    variant: 'cyr',
+    runforEnv: ['local', 'test', 'live'],
+    tests,
+  },
+];
 
-      urls?.forEach(currentPath => {
-        describe(`${pageType} - ${envs.baseUrl}${currentPath}`, () => {
-          before(() => {
-            Cypress.env('currentPath', currentPath);
-            visitPage(currentPath, pageType);
-          });
-          crossPlatformTests({
-            service,
-            pageType,
-            variant,
-          });
-          testsForCanonicalOnly({
-            service,
-            pageType,
-            variant,
-          });
-        });
-      });
+const ampTestSuites = testSuites.map(testSuite => {
```

Comment:
Should we also consider adding .lite test suites, now that it is supported for Most Read pages?